frontend/app/equity/dividend_computations/[id]/page.tsx

```
@@ -0,0 +1,328 @@
+    // Load computation data from localStorage for demo
+    const computationId = parseInt(params.id as string);
+    const demoComputations = JSON.parse(localStorage.getItem('demoComputations') || '[]');
+    const foundComputation = demoComputations.find((comp: any) => comp.id === computationId);
+    
+    if (foundComputation) {
+      setComputation(foundComputation);
+    }
+  }, [params.id]);
+
+  const handleConfirm = async () => {
+    if (confirm('Are you sure you want to confirm this computation? This action cannot be undone.')) {
```

ESLint

[error] Unexpected confirm.

(no-alert)